rt-cli/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+authors = ["Scott Chamberlain", "Josiah Parry"]
+name = "rt-cli"
+version = "0.1.1"
+edition = "2021"
+license = "MIT"
+
+[dependencies]
+argh = "0.1.12"
+glob = "0.3.1"
+regex = "1.10.6"
+anyhow = "*"
```

Comment:
Curious why any version for this package?

Reply:
Two reasons: 
1. laziness
2. that way it can use the same version of rt-lib when it compiles. 

I will set it to be the same

Reply:
got it